<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>버튼1</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="radio1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>체크박스1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>체크박스2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>체크박스3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
